doc/introduction.rst

```diff
@@ -10,501 +10,327 @@ Introduction
-the object: in IPython or Jupyter you can do::
+2. (For French readers) `An introduction to cognitive neuroscience <https://psy3018.github.io/intro.html>`_ given at the University of Montréal.
 
-    In [3]: SVC?
-    ...
-    Parameters
-    ----------
-    C : float or None, optional (default=None)
-        Penalty parameter C of the error term. If None then C is set
-        to n_samples.
+3. The documentation of ``scikit-learn`` explains each method with tips on practical use and examples: :sklearn:`\ `.  While not specific to neuroimaging, it is often a recommended read.
 
-    kernel : string, optional (default='rbf')
-        Specifies the kernel type to be used in the algorithm.
-        It must be one of 'linear', 'poly', 'rbf', 'sigmoid', 'precomputed'.
-        If none is given, 'rbf' will be used.
-    ...
+4. (For Python beginners) A quick and gentle introduction to scientific computing with Python with the `scipy lecture notes <http://scipy-lectures.github.io/>`_.
+Moreover, you can use ``nilearn`` with `Jupyter <http://jupyter.org>`_ notebooks or
+`IPython <http://ipython.org>`_ sessions. They provide an interactive 
+environment that greatly facilitates debugging and visualisation.
 
-Once the object is created, you can fit it on data. For instance, here we
-use a hand-written digits dataset, which comes with scikit-learn::
 
-    >>> from sklearn import datasets
-    >>> digits = datasets.load_digits()
-    >>> data = digits.data
-    >>> labels = digits.target
+Besides, you can find help on :neurostars:`neurostars <>` for questions
+related to ``nilearn`` and to computational neuroscience in general.
+Finally, the ``nilearn`` team organizes weekly :ref:`office hours <quickstart>`.
+We can also be reached on :ref:`github <https://github.com/nilearn/nilearn/issues>`
+in case you find a bug.
 
-Let's use all but the last 10 samples to train the SVC::
 
-    >>> svc.fit(data[:-10], labels[:-10])   # doctest: +ELLIPSIS
-    SVC(C=1.0, ...)
+Machine learning applications to Neuroimaging
+=============================================
 
-and try predicting the labels on the left-out data::
+``nilearn`` brings easy-to-use machine learning tools that can be leveraged to solve more complex applications.
+The interested reader can dive into the following articles for more content.
 
-    >>> svc.predict(data[-10:])     # doctest: +SKIP
-    array([5, 4, 8, 8, 4, 9, 0, 8, 9, 8])
-    >>> labels[-10:]    # The actual labels
-    array([5, 4, 8, 8, 4, 9, 0, 8, 9, 8])
+We give a non-exhaustive list of such important applications.
+    
+**Diagnosis and prognosis**
 
-To find out more, try the
-:sklearn:`scikit-learn tutorials <tutorial/index.html>`.
+Predicting a clinical score or even treatment response
+from brain imaging with :ref:`supervised
+learning <decoding>` e.g. `[Mourao-Miranda 2012]
+<http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0029482>`_
 
-Finding help
--------------
+**Measuring generalization scores**
```

Comment:
This part sounds a bit weird.

Reply:
Agreed ; I got rid of the title altogether.